src/components/Integrations/BigQuery/AuthorizationPage.js

```diff
@@ -59,23 +60,118 @@ const AuthorizationLayout = ({ dependencies: { bigQueryClient, dopplerUserApiCli
         actionUrl={_('big_query.upgrade_plan_url')}
         logoUrl={bigQueryLogo}
         previewUrl={dataStudioGif}
+        caption={<FormattedMessageMarkdown id="big_query.free_text_data_studio_MD" />}
       />
     );
   }
-
   return (
-    <div className="dp-container">
-      <div className="dp-rowflex p-t-48">
-        <div className="col-lg-5">
-          <AuthorizationForm emails={data} onSubmit={onSubmit} />
+    <div data-active-menu="35" className="dp-library">
+      <div className="dp-container">
+        <div className="dp-rowflex">
+          <div className="col-sm-12 m-t-24">
+            <header className="m-b-30">
+              <a
+                href="index.html"
+                className="dp-button button-medium secondary-green button--round"
+              >
+                {' '}
+                Back{' '}
+              </a>
+            </header>
+          </div>
         </div>
       </div>
+      <header className="hero-banner">
+        <div className="dp-container">
+          <div className="dp-rowflex">
+            <div className="col-sm-12 col-md-12 col-lg-12">
+              <nav className="dp-breadcrumb">
+                <ul>
+                  <li>
+                    <a href={_('big_query.url_legacy_doppler')}>{_('big_query.plus_beginning')}</a>
+                  </li>
+                  <li>
+                    <a href={_('big_query.url_legacy_control_panel')}>
+                      {_('big_query.plus_configuration')}
+                    </a>
+                  </li>
+                  <li>
+                    <a href={_('big_query.url_legacy_control_panel')}>
+                      {_('big_query.plus_control_panel')}
+                    </a>
+                  </li>
+                  <li>{_('big_query.plus_big_query')}</li>
+                </ul>
+              </nav>
+              <h2>{_('big_query.plus_title')}</h2>
+            </div>
+            <div className="col-lg-7 col-md-7 col-sm-12">
+              <FormattedMessageMarkdown id="big_query.plus_header_MD" />
+            </div>
+          </div>
+          <span className="arrow"></span>
+        </div>
+      </header>
+      <section className="dp-container m-t-60">
+        <div className="dp-rowflex">
+          <div className="col-lg-6 col-md-12 col-sm-12 m-b-24 p-b-42">
+            <div className="dp-content-left">
+              <span>{_('big_query.plus_step_one')}</span>
+              <strong>
+                <div className="p-heading">
+                  <FormattedMessageMarkdown id="big_query.plus_body_step_one_MD" />
+                </div>
+              </strong>
+              <p className="m-t-24">{_('big_query.plus_step_one_paragraph')}</p>
+              <div className="m-t-24">
+                <FormattedMessageMarkdown id="big_query.plus_step_one_paragraph_MD" />
+              </div>
+            </div>
+            <AuthorizationForm emails={data} onSubmit={onSubmit} />
```

Comment:
the form should go inside the dp-content-left class. That's why it looks like this:
![image](https://user-images.githubusercontent.com/84402180/132222354-2bf13fcf-61bd-4f03-9a6f-ecaafb12ef29.png)

Reply:
suggest accepted and applied